flowcraft/templates/process_tsv.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+"""
+Purpose
+-------
+This module is intended to process the output in tsv
+ to generate a report in json format.
+Expected input
+--------------
+The following variables are expected whether using NextFlow or the
+:py:func:`main` executor.
+- ``sample_id`` : Sample Identification string.
+- ``tsv``: tsv output.
+Generated output
+----------------
+- ``.report.json``: Data structure for the report
```

Comment:
dotfiles (.report.json, .versions, .status, etc) should not count as generated output. This is reserved to non-hidden files.

Reply:
So the generated output is none?

Reply:
removed this section